### PR TITLE
Docs/Developers/Development_Setup.rst: Update installation section

### PR DESCRIPTION
**Pull request by OVlad**
Updates 'Installing from Git' section:
This updates the code in the 'Installing from Git' section with the 
proper one.

Closes https://github.com/coala/documentation/issues/364

### CODE REVIEW

File: docs/Developers/Development_Setup.rst [hunk 1]
```diff
@@ -39,7 +39,13 @@ master branch from and all of its dependencies with pip using
...
 ::
 
-    (venv)$ pip3 install coala-bears --pre
+    (venv)$ git clone https://github.com/coala/coala
+    (venv)$ cd coala
+    (venv)$ pip3 install .
+    (venv)$ cd -
+    (venv)$ git clone https://github.com/coala/coala-bears
+    (venv)$ cd coala-bears
+    (venv)$ pip3 install .
```
Comment (sils) [inline docs/Developers/Development_Setup.rst]: IMO the paragraph below should be modified as well, explaining to use `-e` at the previous instal commands, you don't want to install it uneditable and then editable, thats not too good for your virtualenv

Comment (OVlad) [inline docs/Developers/Development_Setup.rst]: Ooh. I think I got it.
Do you want me to change the above _pip3_ lines with `pip3 install -e . ` or do you want me to create a paragraph below in which the user will be informed that he needs to add the `-e` flag in order to editable install **coala** from repository?

**Comment by sils** [comment]
nice, looks good, can you rebase it?

**Comment by sils** [comment]
ack https://github.com/coala/coala/pull/3311/commits/3cc1ab9293f77e388026a45ee3903d7a19e37fcc

**Comment by sils** [comment]
reack https://github.com/coala/coala/pull/3311/commits/c1f974a02187dd0fa434a6e75cf1e1298715c16c

**Comment by sils** [comment]
@rultor merge

**Pull request merged**